### PR TITLE
ELEMENTS-1665: fix error/validation message when adding permission

### PR DESCRIPTION
**Pull request by alokhyland**

### CODE REVIEW

**Review by swarnadipa-dev — DISMISSED** [review]

File: ui/nuxeo-document-permissions/nuxeo-popup-permission.js [hunk 1]
```diff
@@ -284,6 +288,12 @@ import '../nuxeo-button-styles.js';
       this._doSend(true);
     }
 
+    doCancel() {
+      // get called when clicked on cancel button
```
Comment (rakeshkumar1019) [inline ui/nuxeo-document-permissions/nuxeo-popup-permission.js]: No need to include comments when the code is self-explanatory.

File: ui/widgets/nuxeo-user-suggestion.js [hunk 2]
```diff
@@ -232,6 +232,14 @@ import './nuxeo-user-group-formatter.js';
          * Results filtering function (optional).
          */
         queryResultsFilter: Function,
+
+        /**
+         * Set to `true` for invalid.
+         */
+        invalid: {
```
Comment (rakeshkumar1019) [inline ui/widgets/nuxeo-user-suggestion.js]: is this discussed with PO that we exposing a new property ?

Comment (alokhyland) [reply ui/widgets/nuxeo-user-suggestion.js]: removed this changes as its not required

File: ui/nuxeo-document-permissions/nuxeo-popup-permission.js (hunk 1 above)
Comment (alokhyland) [reply ui/nuxeo-document-permissions/nuxeo-popup-permission.js]: removed comment

**Review by swarnadipa-dev — APPROVED** [review]

**Review by rakeshkumar1019 — APPROVED** [review]

**Review by arupmmi07 — APPROVED** [review]

**Pull request merged**